swap_meet/clothing.py

```diff
@@ -0,0 +1,13 @@
+from .item import Item
+
+
+class Clothing(Item):
+    def __init__(self, condition = None, age = None):
+        category = "Clothing"
+        super().__init__(category, condition, age)
```

Comment:
You could just do this:

```suggestion
        super().__init__(category= "Clothing", condition, age)
```